libqtile/command/base.py

```diff
@@ -83,7 +83,7 @@ def select(self, selectors: List[SelectorType]) -> CommandObject:
             if items is not None and selector and selector not in items:
                 raise SelectError("", name, selectors)
 
-            obj = obj._select(name, selector)
+            obj = obj._select(name, selector)  # type: ignore
```

Comment:
Could the `type: ignore` be dropped if you annotate `obj: CommandObject = self` above? It might need a bit more tweaks, like doing:
```
maybe_obj = obj._select(name, selector)
if maybe_obj is None:
    ...
obj = maybe_obj
```

Reply:
That little hack worked, thanks